### ISSUE TITLE
Anyone can share  a segnet pretrained model ?

### ISSUE THREAD

**Issue by huangyaling** [yassouali/pytorch_segmentation#92]

**Comment by yassouali** [comment]
Hi @huangyaling 

Unfortunately, right now I don't have a pretrained network. Let's keep the issue open, if I have sometime, I might train one.
If you did train your network, please feel free to propose a pull request and adding it to the pretrained networks.

**Comment by huangyaling** [comment]
I have pretrained one with good result on gpus.but it inference no accuracy on cpu.
emm...i found the weight of batchnorm's value have changed before and after load_state_dict.
Has anyone encounter the same problem?

**Comment by yassouali** [comment]
can you make sure the model is in eval mode when doing the evaluation ? where the model is not in eval mode, the behavior of batch norm changes.

**Comment by huangyaling** [comment]
yeah...i'm sure. it happens when encoder init with nn.Sequential(), but i don't  find why go wrong. maybe, It relate to the version of pytorch.
I use below samples instead of that to circumvent this problem, and it works correctly now.
------------------------------------------------------------------------------------------------------------------------------------
self.stage1_encoder_0=nn.Conv2d(3, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))   
self.stage1_encoder_1=nn.BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
self.stage1_encoder_2=nn.ReLU(inplace=True)                                                 
self.stage1_encoder_3=nn.Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))  
self.stage1_encoder_4=nn.BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
 self.stage1_encoder_5=nn.ReLU(inplace=True)
----------------------------------------------------------------------------------------------------------------------------------------
thanks for your help~~~

**Comment by gauridhawan** [comment]
@huangyaling what's the value of meanIoU after making these changes? I'm getting 0.3 rn
